commandline/src/main/java/tech/pegasys/web3signer/commandline/valueprovider/EnvironmentVariableDefaultProvider.java

```diff
@@ -33,9 +33,15 @@ public String defaultValue(final ArgSpec argSpec) {
       final OptionSpec optionSpec = (OptionSpec) argSpec;
       final String qualifiedPrefix =
           optionSpec.command().qualifiedName("_").replace("-", "_").toUpperCase();
-      final String key = stripPrefix(optionSpec.longestName()).replace("-", "_").toUpperCase();
-      final String qualifiedKey = qualifiedPrefix + "_" + key;
-      return environment.get(qualifiedKey);
+
```

Comment:
I think this is also coming through your other PR ... might cause merge conflict.